csharp/Platform.Setters/Setter[TResult].cs

```diff
@@ -5,32 +5,26 @@
 namespace Platform.Setters
 {
     /// <summary>
-    /// <para>
-    /// Represents the setter.
-    /// </para>
-    /// <para></para>
+    /// <para>Represents the setter.</para>
+    /// <para>Представляет установщик.</para>
     /// </summary>
     /// <seealso cref="Setter{TResult, bool}"/>
```

Comment:
Remove this

Reply:
No need to remove reference to class with other number of type arguments. It can be useful to give a user ability to explore all available implementations.